installing/installing_alibaba/manually-creating-alibaba-ram.adoc

```diff
@@ -0,0 +1,19 @@
+[id="manually-creating-alibaba-ram"]
+= Manually creating IAM for Alibaba Cloud
```

Comment:
not sure about "Resources", wdyt @joelddiaz @kwoodson ?

Reply:
maybe "RAM credentials"?

Reply:
They are resources. An AliCloud User is a RAM Resource (using AliCloud terminology). If the User is not a Resource, then what exactly is it?

Reply:
I am thinking about this a bit differently. We are putting the CCO into manual mode because the user must manage the cloud credentials, instead of the CCO. From a user-facing perspective, that is what is relevant. In a similar section for the IPI work I am covering for Azure Stack Hub, I went with Manually manage cloud credentials [1].

[1] [Manually manage cloud credentials](https://deploy-preview-39366--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_azure_stack_hub/installing-azure-stack-hub-default.html#manually-create-iam_installing-azure-stack-hub-default)

Reply:
@mjpytlak "Manually manage cloud credentials" looks good to me.